me.eastsun.hackdesk.yml

```diff
@@ -0,0 +1,24 @@
+app-id: me.eastsun.hackdesk
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: hackdesk
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland # Permission needed to show the window
+  - --socket=x11 # Permission needed to show the window
+  - --socket=fallback-x11
```

Comment:
it is NEVER all three together. Either `x11` alone or `wayland` + `fallback-x11` depending on what the app supports.

Reply:
Fixed! I leave `x11`